tailnet/tunnel.go

```diff
@@ -1,9 +1,16 @@
 package tailnet
 
-import "github.com/google/uuid"
+import (
+	"net/netip"
+
+	"github.com/google/uuid"
+)
+
+var legacyWorkspaceAgentIP = netip.MustParseAddr("fd7a:115c:a1e0:49d6:b259:b7ac:b1b2:48f4")
 
 type TunnelAuth interface {
 	Authorize(dst uuid.UUID) bool
+	AuthorizeIP(src uuid.UUID, ip netip.Prefix) bool
```

Comment:
We know the `src` when the `TunnelAuth` is instantiated (i.e. the peer connects), so it shouldn't be a parameter on the call. It should be a field in the agent auth struct, not needed in client (for now).